### ISSUE TITLE
 $digest already in progress?

### ISSUE THREAD

**Issue by mikehuebner** [zurb/foundation-apps#393]
I keep getting this error with the Notifications, wondered if this was known or an issue I caused. Thanks!

Error:
![image](https://cloud.githubusercontent.com/assets/6125123/5672997/59296f2c-974b-11e4-9143-65a6ee5c0992.png)

Code that opens it:
```
FoundationApi.publish('login-fail', {
title: 'Login Failed!',
content: user.msg,
color: 'alert'
});
```

**Comment by jeaniec** [comment]
I'll look into this, thanks for catching it!

**Comment by AntJanus** [comment]
There were some fixes that went out the door a few weeks back, and one specifically regarding this issue, are you on v1.0.2?

**Comment by mikehuebner** [comment]
Yea I'm on latest!

**Comment by jeaniec** [comment]
Fixed with 23fee8743590058f4b084393fd15d593c54d8e7e
